Example/Tests/GiniVisionDocumentTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  GINIFilePickerManagerTests.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 8/30/17.
+//  Copyright © 2017 CocoaPods. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision
+
+
+class GINIVisionDocumentTests: XCTestCase {
+    
+    var filePickerManager:FilePickerManager {
+        return FilePickerManager()
+    }
+    
+    func testExcedeedMaxFileSize() {
+        // Given
+        let higherThan10MBData = generateFakeData(megaBytes: 12)
+        
+        // When
+        let fakeDocument = GiniPDFDocument(data: higherThan10MBData)
+        
+        // Then
+        XCTAssertThrowsError(try fakeDocument.validate(), "should indicate that is not a valid document") { error in
+            XCTAssert(error as? DocumentValidationError == DocumentValidationError.exceededMaxFileSize, "should indicate that max file size has been exceeded")
+        }
+    }
+    
+    func testNotExcedeedMaxFileSize() {
+        // Given
+        let lowerThanOrEqualTo10MBData = generateFakeData(megaBytes: 10)
+        
+        // When
+        let fakeDocument = GiniPDFDocument(data: lowerThanOrEqualTo10MBData)
+
+        // Then
+        XCTAssertThrowsError(try fakeDocument.validate(), "should indicate that is not a valid document") { error in
+            XCTAssert(error as? DocumentValidationError != DocumentValidationError.exceededMaxFileSize, "should indicate that max file size has been exceeded")
+        }
+    }
+    
+    func testImageValidation() {
+        // Given
+        let testBundle = Bundle(for: type(of: self))
+        let image = UIImage(named: "tabBarIconHelp", in: testBundle, compatibleWith: nil)
```

Comment:
I ran into the same thing in the Switch SDK. Getting an image from the framework's bundle is annoying (since you always have to remember to get it from the framework's bundle, not the main one). Can you extract it in a utility method/class so you don't have to repeat yourself in the future?

Reply:
It could be useful 👌